Mathlib/CategoryTheory/Sites/Equivalence.lean

```diff
@@ -64,22 +64,44 @@ theorem locallyCoverDense : LocallyCoverDense J e.inverse := by
       exact T.val.downward_closed hf _
     · simp
 
-theorem coverPreserving : CoverPreserving J (e.locallyCoverDense J).inducedTopology e.functor where
+theorem functorPushforward_eq_pullback {U : C} (S : Sieve U) :
```

Comment:
I would think that this lemma (and the next) should be moved to a more basic file. Otherwise, the PR looks good to me.